codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AwsSmithyRpcV2Cbor.java

```diff
@@ -55,15 +56,18 @@ protected void writeSharedRequestHeaders(ProtocolGenerator.GenerationContext con
 
     @Override
     protected void writeErrorCodeParser(GenerationContext generationContext) {
-        super.writeErrorCodeParser(generationContext);
```

Comment:
caller super here put the lines in the wrong order previously

Reply:
there's no codegen diff for this since the protocol test for cbor does not run with awsQueryCompat on